jck/jck.mk

```diff
@@ -65,6 +61,11 @@ endif
 
 SYSTEMTEST_RESROOT=$(TEST_RESROOT)/../../system
 
+# If no testsuite specified, use RUNTIME
+ifeq (,$(findstring testsuite, $(CUSTOM_TARGET)))
+   CUSTOM_TARGET:=$(CUSTOM_TARGET),testsuite=RUNTIME
```

Comment:
I think this should be JCK_CUSTOM_TARGET, not CUSTOM_TARGET

Reply:
Worked with Lan over slack to get this fixed. 

Yes, we did move back to using JCK_CUSTOM_TARGET but had to use `override` to append the default testsuite to it. 

Tests: 
~~~
Case 1) Testsuite provided by user: Grinder/15659
Case 2) Testsuite is not provided: Grinder/15661 (Note 1 / 23 test failure is unrelated to this PR - it's just a tck test failure)

Reply:
Thanks @Mesbah-Alam